server/src/handlers/file_handler.rs

```diff
@@ -181,6 +182,17 @@ pub async fn upload_file_handler(
 
     bucket_upload_span.finish();
 
+    let file_size_mb = (decoded_file_data.len() as f64 / 1024.0 / 1024.0).round() as i64;
+
+    create_file_query(
+        file_id,
+        file_size_mb,
+        upload_file_data.clone(),
+        dataset_org_plan_sub.dataset.id,
+        pool.clone(),
+    )
+    .await?;
+
```

Comment:
Seems odd that this would need to get added? Is there a reason here?

Reply:
Bc before it would've been done in the worker, in this case its being sent to the microservice, so we still need to make sure that the file is created in our PG